### PR TITLE
Replaced obsolete URL https://mirror.openshift.com/pub/openshift-v4/o…

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
…c/latest with the new URL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/

Per: 
https://github.com/openshift/openshift-docs/issues/30745#issuecomment-804882784

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
Please note that the original issue, https://github.com/openshift/openshift-docs/issues/30745 , lists three web URL's needing corrected. HOWEVER, OpenShift Docs team did a terrific job on the Vsphere installation section and re-used the same module across all three installation files. Therefore only one commit and one PR https://github.com/openshift/openshift-docs/pull/30831 is needed to correct this issue!

File: modules/cli-installing-cli.adoc [hunk 1]
```diff
@@ -60,7 +60,7 @@ You can install the OpenShift CLI (`oc`) binary on Linux by using the following
 .Procedure
 
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your operating system and architecture.
```
Comment (bobfuru) [inline modules/cli-installing-cli.adoc:63]: Using this URL seems fine to me but we should probably clarify the wording a bit because it lands on a page of OCP versions now. How about something like this?
`. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your {product-title} version, operating system, and architecture.`
Seems less to maintain than using the release-specific URL and having to update the procedure with each release. :)

**Comment by GroceryBoyJr** [comment]
This URL correction is to a module used in the following display pages:
https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html
https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html
https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html

**Review by bobfuru — COMMENTED** [review]
A suggestion for wording to correspond to the new URL. Also, you might want to shorten the title and edit the trailing text in comment 0 (GH generates the PR title from the commit message). Thanks!!

File: modules/cli-installing-cli.adoc [hunk 2]
```diff
@@ -98,7 +98,7 @@ You can install the OpenShift CLI (`oc`) binary on Windows by using the followin
...
```
Comment (bergerhoffer) [inline modules/cli-installing-cli.adoc:101]: This procedure is actually for installing on Windows, so this x86_64 link (which is for Linux), isn't appropriate.

And looking at the deprecation notice in that original location, only the x86_64 version for the Linux procedure is deprecated. So I think you can revert this update completely.

File: modules/cli-installing-cli.adoc [hunk 3]
```diff
@@ -131,7 +131,7 @@ You can install the OpenShift CLI (`oc`) binary on macOS by using the following
...
```
Comment (bergerhoffer) [inline modules/cli-installing-cli.adoc:134]: This procedure is actually for installing on macOS, so this x86_64 link (which is for Linux), isn't appropriate.

And looking at the deprecation notice in that original location, only the x86_64 version for the Linux procedure is deprecated. So I think you can revert this update completely.

File: modules/cli-installing-cli.adoc (hunk 1 above)
Comment (bergerhoffer) [inline modules/cli-installing-cli.adoc:63]: Although x86_64 is appropriate for Linux, the rest of this step text ("and choose the folder for your operating system and architecture.") doesn't match what follows after you've navigated to this new location. Because you've already chosen the operating system and architecture with this path.

And in this new location, there are a ton of options now, and I'm not positive, but it might be best to point them in the right direction of where to go next (or take them directly into one of the version subfolders).

**Comment by bergerhoffer** [comment]
FYI @GroceryBoyJr @bobfuru - I checked with @vikram-redhat and it looks like we'll only need to cherry pick this to enterprise-4.8. I can't remember if I mentioned the background to both of you, but OKD docs are built directly off the master branch, but we decided to also cherry pick the changes to the latest enterprise branch, because in the past we dealt with too many merge conflicts. So we are now trying to keep master and the latest enterprise branch as in sync as possible.

I'll check whether this info is in the manual, and if not, I'll add it.

Long story short - I've removed the 4.5-4.7 labels :)
